templates/primary-secondary/primary-secondary.js

```diff
@@ -953,6 +953,29 @@ class TemplatePrimarySecondary extends RtlMixin(LocalizeCoreElement(LitElement))
 					z-index: -1;
 				}
 			}
+
+			@media print {
+				:host([print-primary-only]) {
+					overflow: visible;
+					position: relative;
+				}
+
+				:host([print-primary-only]) .d2l-template-primary-secondary-container,
+				:host([print-primary-only]) .d2l-template-primary-secondary-content {
+					height: auto;
+					overflow: visible;
+				}
+
+				:host([print-primary-only]) .d2l-template-primary-secondary-container > header,
+				:host([print-primary-only]) .d2l-template-primary-secondary-container > footer,
+				:host([print-primary-only]) .d2l-template-primary-secondary-content > :not(main) {
+					display: none;
+				}
+
+				:host([print-primary-only]) .d2l-template-primary-secondary-content > main {
+					background: white;
+				}
+			}
```

Comment:
I think we want a lot of these changes (showing overflow, turning off the background colours) no matter what? I doubt much thought was ever put in to how this prints out when this template was built.

For the `print-primary-only` thing: I could see this get really complicated if folks want the primary and secondary but not the header/footer, or the header and primary but not the secondary/footer, and so on. Instead of the attribute, could you include CSS in your app to hide the stuff you're putting in those slots, or does that not work?